taxi/views.py

```diff
@@ -34,12 +35,52 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `ManufacturerCreateView` exposes all fields of the model. Verify that this is intentional and that no sensitive fields are included. It's often safer to specify only the fields that should be editable.